docs/empower.rst

```diff
@@ -0,0 +1,47 @@
+Empower
+=======
+
+********
+Overview
+********
+
+The Empower class allows you to interact with the Empower API. Documentation for the Empower API can be found
+in their `GitHub <https://github.com/getempower/api-documentation/blob/master/README.md>`_ repo.
+
+.. note::
+    The Empower API only has a single endpoint to access all account data. As such, it has a very high overhead. This
+    connector employs caching in order to allow the user to specify the tables to extract without additional API calls.
+    You can disable caching as an argument when instantiating the class.
+
+==========
+Quickstart
+==========
+
+To instantiate the Empower class, you can either store your ``EMPOWER_API_KEY`` an environment
+variables or pass them in as arguments:
+
+.. code-block:: python
```

Comment:
Might be nice to have an example of disabling caching in the quickstart

Reply:
Can do.